src/training_data.py

```diff
@@ -117,23 +122,35 @@ def load_api_data(self, files):
         for filename in files:
             data = json.loads(codecs.open(filename, encoding='utf-8').read())
             # get only intents, skip the rest. The property name is the target class
-            if "userSays" not in data:
-                continue
-
-            intent = data.get("name")
-            for s in data["userSays"]:
-                text = "".join(map(lambda chunk: chunk["text"], s.get("data")))
-                # add entities to each token, if available
-                entities = []
-                for e in filter(lambda chunk: "alias" in chunk or "meta" in chunk, s.get("data")):
-                    start = text.find(e["text"])
-                    end = start + len(e["text"])
-                    val = text[start:end]
-                    entities.append(
-                        {"entity": e["alias"] if "alias" in e else e["meta"], "value": val, "start": start, "end": end})
-
-                self.intent_examples.append({"text": text, "intent": intent})
-                self.entity_examples.append({"text": text, "intent": intent, "entities": entities})
+            if "userSays" in data:
+                intent = data.get("name")
+                for s in data["userSays"]:
+                    text = "".join(map(lambda chunk: chunk["text"], s.get("data")))
+                    # add entities to each token, if available
+                    entities = []
+                    for e in filter(lambda chunk: "alias" in chunk or "meta" in chunk, s.get("data")):
+                        start = text.find(e["text"])
+                        end = start + len(e["text"])
+                        val = text[start:end]
+                        entities.append(
+                            {
+                                "entity": e["alias"] if "alias" in e else e["meta"],
+                                "value": val,
+                                "start": start,
+                                "end": end
+                            }
+                        )
+
+                    self.intent_examples.append({"text": text, "intent": intent})
+                    self.entity_examples.append({"text": text, "intent": intent, "entities": entities})
+
+            # create synonyms dictionary
```

Comment:
don't have a strong opinion on this one. i am just worried about the length of this function.